libs/features/mas/web-components/src/global.css.js

```diff
@@ -138,7 +138,7 @@ merch-card span[is=inline-price] {
 }
 
 merch-card [slot='heading-xs'] {
-    color: var(--merch-color-grey-80);
+    color: var(--spectrum-gray-800, #2c2c2c);
```

Comment:
It appears that is not the case in catalog page: https://main--cc--adobecom.hlx.page/products/catalog?milolibs=mwpw-158200--milo--adobecom
![Screenshot 2024-10-07 at 9 59 23 AM](https://github.com/user-attachments/assets/2b2f2aea-b953-4a9c-98f6-01754f0268fb)
And even if the spectrum styles were defined, it should be okay since we don't use dark theme in Milo and the colors are basically the same. I don't mind writing the conditional logic, but I think we should first justify the need of it.

thoughts @3ch023 @npeltier ?

Reply:
Thanks for checking.
At least can we avoid using inline values for colors and use color 
and do `var(--spectrum-gray-800, var(--merch-color-grey-80));` ?

Reply:
personally, I don't see any value in keeping repeated hard-coded values.
if neither of `--spectrum-gray-800` or `--merch-color-grey-80` is resolved, we probably have a bigger problem.

Reply:
You bring a good point. I have changed it to `var(--spectrum-gray-800, var(--merch-color-grey-80`.